src/main/java/racingcar/view/RacingGameInputView.java

```diff
@@ -0,0 +1,22 @@
+package racingcar.view;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.Arrays;
+
+public class RacingGameInputView {
+    private static final String DELIMITER = ",";
+
+    public static String next() {
+        return Console.readLine();
+    }
+
+    public static Integer nextInt() {
+        return Integer.parseInt(next());
```

Comment:
숫자가 아닌 값이 들어올 때를 대비해서 exception 을 변환해야 할 것 같습니다.
```suggestion
        try {
            return Integer.parseInt(next());
        } catch (NumberFormatException e) {
            throw new IllegalArgumentException("not a number", e);
        }
```

Reply:
오 그러네요 숫자에 대한 검증 인사이트 얻어갑니다 예시 작성 감사합니다 !! 👍